pkgs/development/compilers/uasm/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "uasm";
+  version = "2.53";
+
+  src = fetchFromGitHub {
+    owner = "Terraspace";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Aohwrcb/KTKUFFpfmqVDPNjJh1dMYSNnBJ2eFaP20pM=";
+  };
+
+  patches = [
+    # fix `dbgcv.c:*:*: fatal error: direct.h: No such file or directory`
+    (fetchpatch {
+      name = "fix_build_dbgcv_c_on_unix.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/Terraspace/UASM/pull/154.patch";
```

Comment:
That isn't merged yet so we need to fetch each commit separate.

Reply:
Fixed.